app.js

```diff
@@ -7,15 +7,58 @@ client.on('ready', () => {
 });
 client.login(process.env.BOT_TOKEN)
 
+async function setup(msg) {
+    //https://greenwichmeantime.com/time-zone/definition/
+    //https://github.com/ac360/currency-codes-array-ISO4217
+    const timezoneCodes = ['IDLW', 'NT', 'HST', 'AKST', 'PST', 'MST', 'CST', 'EST', 'AST', 'ART', 'AT', 'WAT', 'GMT', 'CET', 'EET', 'MSK', 'AMT', 'PKT', 'OMSK', 'KRAT', 'CST', 'JST', 'AEST', 'SAKT', 'NZST'];
+    const currencyCodes = ['AED', 'AFN', 'ALL', 'AMD', 'ANG', 'AOA', 'ARS', 'AUD', 'AWG', 'AZN', 'BAM', 'BBD', 'BDT', 'BGN', 'BHD', 'BIF', 'BMD', 'BND', 'BOB', 'BOV', 'BRL', 'BSD', 'BTN', 'BWP', 'BYR', 'BZD', 'CAD', 'CDF', 'CHE', 'CHF', 'CHW', 'CLF', 'CLP', 'CNY', 'COP', 'COU', 'CRC', 'CUC', 'CUP', 'CVE', 'CZK', 'DJF', 'DKK', 'DOP', 'DZD', 'EGP', 'ERN', 'ETB', 'EUR', 'FJD', 'FKP', 'GBP', 'GEL', 'GHS', 'GIP', 'GMD', 'GNF', 'GTQ', 'GYD', 'HKD', 'HNL', 'HRK', 'HTG', 'HUF', 'IDR', 'ILS', 'INR', 'IQD', 'IRR', 'ISK', 'JMD', 'JOD', 'JPY', 'KES', 'KGS', 'KHR', 'KMF', 'KPW', 'KRW', 'KWD', 'KYD', 'KZT', 'LAK', 'LBP', 'LKR', 'LRD', 'LSL', 'LTL', 'LVL', 'LYD', 'MAD', 'MDL', 'MGA', 'MKD', 'MMK', 'MNT', 'MOP', 'MRO', 'MUR', 'MVR', 'MWK', 'MXN', 'MXV', 'MYR', 'MZN', 'NAD', 'NGN', 'NIO', 'NOK', 'NPR', 'NZD', 'OMR', 'PAB', 'PEN', 'PGK', 'PHP', 'PKR', 'PLN', 'PYG', 'QAR', 'RON', 'RSD', 'RUB', 'RWF', 'SAR', 'SBD', 'SCR', 'SDG', 'SEK', 'SGD', 'SHP', 'SLL', 'SOS', 'SRD', 'SSP', 'STD', 'SYP', 'SZL', 'THB', 'TJS', 'TMT', 'TND', 'TOP', 'TRY', 'TTD', 'TWD', 'TZS', 'UAH', 'UGX', 'USD', 'USN', 'USS', 'UYI', 'UYU', 'UZS', 'VEF', 'VND', 'VUV', 'WST', 'XAF', 'XAG', 'XAU', 'XBA', 'XBB', 'XBC', 'XBD', 'XCD', 'XDR', 'XFU', 'XOF', 'XPD', 'XPF', 'XPT', 'XTS', 'XXX', 'YER', 'ZAR', 'ZMW'];
+    const content = msg.content.slice(7).toUpperCase().trim().split(' ');  //7 is length of '!setup '
+    const timezone = content[0];
+    const currency = content[1];
+    let timezoneFound = false;
+    let currencyFound = false;
+    timezoneCodes.forEach(item => {
+        if(item === timezone){
+            timezoneFound = true;
+        }});
+    currencyCodes.forEach(item => {
+        if(item === currency){
+            currencyFound = true;
+        }});
```

Comment:
Just one suggestion, can you replace the forEach with `currencyCodes.includes(currency)`?

i know I recommended the .forEach, i was imagining checking the other way (like: does each item match any of the array elements)

Now that we have the established format for setup messages, we don't need that, so you did this well

Reply:
you can even just do `const currencyFound = currencyCodes.includes(currency);`

Reply:
Ah good call!